### ISSUE TITLE
Error updating state after manually deleting certificate

### ISSUE THREAD

**Issue by ryansch** [fly-apps/terraform-provider-fly#98]
Repro:
1. Use the fly terraform provider to create a certificate via `resource "fly_cert"`.
2. Manually remove the certificate from fly.
3. `terraform plan`
4. Observe the following error.

Error: Could not find AppCertificate
app.certificate

I was able to work around this by using `terraform state rm`.

**Comment by DAlperin** [comment]
@ryansch yeah that is a terraform thing more than an us thing. Modifying state outside of terraform is always bound to cause conflicts. I'll see if I can't make it spit out a better error message for this.

**Comment by mootari** [comment]
Shouldn't the resource get recreated in that case though, instead of just throwing an error?
